third_party/ijar/zip.cc

```diff
@@ -893,7 +894,7 @@ void OutputZipFile::WriteCentralDirectory() {
   for (size_t ii = 0; ii < entries_.size(); ++ii) {
     LocalFileEntry *entry = entries_[ii];
     put_u4le(q, CENTRAL_FILE_HEADER_SIGNATURE);
-    put_u2le(q, 0);  // version made by
+    put_u2le(q, UNIX_ZIP_FILE_VERSION);
```

Comment:
The end of central directory record should also be updated to include the version. Line 928.

Reply:
Thanks for catching that! I was looking at the ZIP spec and completely missed the ZIP64 extension!